SimpleAudioPlayer/Teddy.MvvmCross.Plugins.SimpleAudioPlayer/MvxSimpleAudioPlayerExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Teddy.MvvmCross.Plugins.SimpleAudioPlayer
+{
+    public static class SimpleAudioPlayerExtensions
+    {
+        public static void PlayOnRepeat(this IMvxSimpleAudioPlayer player)
+        {
+            player.Completion += new EventHandler(delegate (Object o, EventArgs a)
```

Comment:
We should not add the EventHandler to Completion event on every call, otherwise there might be duplicated events be called. At least, please add a flag to store the "event handler registered state". And when we have a "player.Completion +=", we should also have a "player.Completion -=" somewhere to be able to reset the Repeat function when not necessary anymore. This is especially important for UWP, cause we are using a shared singleton instance of player.